### PR TITLE
Add test OCA\Files_Trashbin\Capabilities

### PR DESCRIPTION
**Pull request by forsd**

### CODE REVIEW

**Comment by MorrisJobke** [comment]
Hey @forsd 

Thanks for your contribution. There is only one thing missing - you need to add a sign-off message to the commits you made. Read more about this in https://github.com/nextcloud/server/blob/master/.github/CONTRIBUTING.md#sign-your-work


Thanks 🚀

File: apps/files_trashbin/tests/CapabilitiesTest.php [hunk 1]
```diff
@@ -0,0 +1,44 @@
+<?php
+/**
+ * @license GNU AGPL version 3 or any later version
```
Comment (MorrisJobke) [inline apps/files_trashbin/tests/CapabilitiesTest.php]: Is a copyright owner actually needed? cc @schiessle

**Review by MorrisJobke — APPROVED** [review]
Beside my comment the code looks good 👍

File: apps/files_trashbin/tests/CapabilitiesTest.php (hunk 1 above)
Comment (rullzer) [reply apps/files_trashbin/tests/CapabilitiesTest.php]: Not as far as I know.

**Review by ChristophWurst — APPROVED** [review]
Awesome :v:

**Comment by MorrisJobke** [comment]
Let's merge this as this is not that much code and basically only adds one test case

**Comment by MorrisJobke** [comment]
@forsd Thanks for you contribution. If you are looking for other issues to solve you could have a look at  https://github.com/nextcloud/server/issues?q=is%3Aopen+is%3Aissue+label%3A%22good+first+issue%22

**Pull request merged**